docgen/content-sources/node/toc.yaml

```diff
@@ -178,6 +178,20 @@ toc:
   - title: "ShaCertificate"
     path: /docs/reference/admin/node/admin.projectManagement.ShaCertificate
 
+- title: "admin.securityRules"
```

Comment:
I'll defer to you and your other reviewers here, but we use `FirebaseRules` in the same way we use `FirebaseAuth` and `FirebaseDatabase`. The extra "security" word here breaks conventions. ((Consider: what the object will be called in the Java admin SDK?))

Reply:
Current naming convention used is: `admin.securityRules()`, `SecurityRules`, (Java: `FirebaseSecurityRules`).

It sounds like you're suggesting: `admin.rules()`, `Rules`, (Java: `FirebaseRules`)

Is that correct? I don't have preference. Let's see what @rachelmyers and others think. The existing documentation uses "Firebase Security Rules" pretty much everywhere: https://firebase.google.com/docs/rules